AccessCheckoutSDK/pacts/access-checkout-ios-sdk-sessions.json

```diff
@@ -303,72 +340,73 @@
     {
       "description": "POST request to /sessions/card with valid body",
       "request": {
-        "method": "post",
-        "path": "/sessions/card",
+        "method": "get",
```

Comment:
we seem to have replaced that part of the contract that covers the creation of card session with the part of the contract for service discovery?